libs/coin-modules/coin-solana/src/types.ts

```diff
@@ -0,0 +1,284 @@
+import BigNumber from "bignumber.js";
+import {
+  Account,
+  AccountRaw,
+  Operation,
+  TransactionCommon,
+  TransactionCommonRaw,
+  TransactionStatusCommon,
+  TransactionStatusCommonRaw,
+} from "@ledgerhq/types-live";
+import { ValidatorsAppValidator } from "./validator-app";
+
+export type TransferCommand = {
+  kind: "transfer";
+  sender: string;
+  recipient: string;
+  amount: number;
+  memo?: string;
+};
+
+export type TokenCreateATACommand = {
```

Comment:
Maybe `types.ts` files are also too much like a "fourre tout". I think it could be nice to have things organized by purpose/role.

Like a token.ts with the type, the helper functions, etc

Reply:
I don't want to refacto this coin-module for the time being.
The only purpose is to separate the code from LLC, as we have done for Algorand, Near and Bitcoin.